src/main/java/com/wavefront/opentracing/reporting/WavefrontOpenTracingReporter.java

```diff
@@ -0,0 +1,110 @@
+package com.wavefront.opentracing.reporting;
+
+import com.wavefront.opentracing.Reference;
+import com.wavefront.opentracing.WavefrontSpan;
+import com.wavefront.opentracing.WavefrontSpanContext;
+import com.wavefront.sdk.common.WavefrontSender;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+import java.util.UUID;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+/**
+ * The reporter which reports tracing spans to Wavefront via WavefrontSender.
+ *
+ * @author Vikram Raman (vikram@wavefront.com)
+ */
+public class WavefrontOpenTracingReporter implements Reporter {
```

Comment:
We can just call this `WavefrontReporter` since it is already under `com.wavefront.opentracing.reporting`